snuba/web/views.py

```diff
@@ -435,25 +438,49 @@ def handle_subscription_error(exception: InvalidSubscriptionError) -> Response:
     )
 
 
+@application.route("/<dataset:dataset>/<entity:entity>/subscriptions", methods=["POST"])
 @application.route("/<dataset:dataset>/subscriptions", methods=["POST"])
```

Comment:
Are you planning to remove this one as a follow up? I think we should

Reply:
Yep that's the plan, I am planning to remove line 442 and 470 i.e. for creation and deletion as a follow up

Reply:
Sounds good, it could be useful to mark or comment these in code so it's clear what will happen to the old routes